content/03.prior-knowledge.knowledge-graphs.md

```diff
@@ -1,17 +1,17 @@
+Examples of public biomedical KGs and frameworks that could be useful in rare disease include the Monarch Graph Database[@doi:10.1093/nar/gkw1128], hetionet[@doi:10.7554/eLife.26726], PheKnowLator[@doi:10.1101/2020.04.30.071407], and the Global Network of Biomedical Relationships[@doi:10.1093/bioinformatics/bty114]. 
+These graphs connect information like genetic, functional, chemical, clinical, and ontological data to enable the exploration of relationships of data with disease phenotypes through manual review[@doi:10.1093/database/baaa015] or computational methods[@doi:10.1101/727925; @doi:10.1186/s12911-019-0938-1]. 
 
-An intrinsic constraint in the study of rare disease is the availability of large, normalized datasets. 
-This limits our ability to study genotype-phenotype relationships or other key attributes of rare diseases. 
-A potentially powerful strategy for evaluating genotype-phenotype relationships or repurposing drugs when large datasets are scarce or nonexistent is to develop and use knowledge graphs.
-Knowledge graphs integrate related-but-different data types, creating a rich and complex data source. 
-Examples of well-known public biomedical knowledge graphs and graph frameworks that could be useful in the rare disease context include the Monarch Graph Database[@doi:10.1093/nar/gkw1128], hetionet[@doi:10.7554/eLife.26726], PheKnowLator[@doi:10.1101/2020.04.30.071407], and the Global Network of Biomedical Relationships[@doi:10.1093/bioinformatics/bty114]. 
-These graphs connect information like genetic, functional, chemical, clinical, and ontological data to enable the end user to explore many types of data and their relationships with disease phenotypes whether through manual review[@doi:10.1093/database/baaa015] or computational methods[@doi:10.1101/727925; @doi:10.1186/s12911-019-0938-1]. 
-In the academic rare disease space, there a few pioneering examples of machine learning-based mining of knowledge graphs to repurpose drugs[@doi:10.1101/727925] and classify rare diseases[@doi:10.1186/s12911-019-0938-1].
-These studies make it clear that there are some challenges in using machine learning using graph databases in rare disease.
-For example, these papers rely on a gold standard dataset to validate the performance of the models; often, there are not robust gold standard datasets available for individual rare diseases.
-These methods also evaluate a broad swath of rare diseases in a relatively unguided manner, rather than interrogating a pre-defined disease of interest.
-Consequently, it is not yet clear how effective these approaches, and knowledge graphs in general, are in studying a specific disease of interest; more work needs to be done to identify methods that can provide actionable insights for a specific rare disease application. 
-Beyond the aforementioned studies, there are very few examples of studies in the public domain that leverage knowledge graphs to characterize rare disease. 
```

Comment:
I do feel like this section in particular may have been cut too aggressively.

Reply:
Added back a bit of content here. I'm not sure what is going on with the diff when I first filed the PR, but I didn't change all that much about these lines...